src/js/services/api.js

```diff
@@ -63,6 +78,21 @@ export default (() => {
       id: 'user_course_session_id',
       url: id => `/user_course_sessions/${id}/feedback`
     },
+    {
+      name: 'getInstructors',
+      type: 'GET',
+      url: '/get_instructors'
```

Comment:
url shouldn't have `get` in it. It should be `/instructors` with action `GET`. change in both front end and backend